djgentelella/management/commands/show_permissions.py

```diff
@@ -0,0 +1,68 @@
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from django.contrib.auth.models import Permission
+from djgentelella.models import PermissionRelated
+
+
+
+class Command(BaseCommand):
+    help = 'Create related permissions in the database'
+    permissions_to_create = []
+
+    def import_module_app_gt(self, app, name):
+        try:
+            variable = __import__(app + '.' + name)
+            return variable.gtpermissions.permissions_to_create
+        except ModuleNotFoundError as e:
+            pass
+        except AttributeError as e:
+            print(f"Error de atributo: {str(e)}")
```

Comment:
```suggestion
        except AttributeError as e:
            print(f"Error de atributo: {str(e)}")
```
Use `logger.exception("Attribute error")` instead of print.